plugin/neomake-local-eslint.vim

```diff
@@ -0,0 +1,23 @@
+function! GetNpmBin(binname)
+  let dir = getcwd()
+  while ! isdirectory(dir . '/node_modules')
+    let dir = fnamemodify(dir, ':p:h')
+    if dir == '/'
+      break
+    end
+  endwhile
+
+  let binpath = ''
+  if dir != '/'
+    let binpath = dir . '/node_modules/.bin/' . a:binname
+    if ! filereadable(binpath)
+      let binpath = ''
+    end
+  endif
+
+  if empty(binpath)
+    let binpath = system('echo -n $(npm bin)') . '/' . a:binname
```

Comment:
`echo -n $(...)` used to avoid newline at end, vim doesn't seem to have a `strip`/`chomp` function.